configuration/frameworks/framework_config.cfg.default

```diff
@@ -45,11 +45,11 @@ TARGETS_DIR: targets
 
```

Comment:
Yes, all the profiles should be in configuration folder IMO. It makes no sense as only one config files will be present for each case.